ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -106,18 +106,18 @@ import SwiftUI
 }
 
 /// Properties available to customize the state of the avatar
-class MSFAvatarStateImpl: NSObject, ObservableObject, MSFAvatarState {
-    @Published var image: UIImage?
-    @Published var primaryText: String?
-    @Published var secondaryText: String?
-    @Published var ringColor: UIColor?
-    @Published var backgroundColor: UIColor?
-    @Published var foregroundColor: UIColor?
-    @Published var presence: MSFAvatarPresence = .none
-    @Published var hasPointerInteraction: Bool = false
-    @Published var isRingVisible: Bool = false
-    @Published var isTransparent: Bool = true
-    @Published var isOutOfOffice: Bool = false
+public class MSFAvatarStateImpl: NSObject, ObservableObject, MSFAvatarState {
```

Comment:
This should remain internal and we need to expose it as an MSFAvatarState.
We can create public APIs to create new avatar state internally and return the instance as a reference to the MSFAvatarState protocol.